packages/store/src/StoreCore.sol

```diff
@@ -326,53 +331,53 @@ library StoreCore {
   }
 
   /**
-   * Get a single field from the given table and key tuple (loading schema from storage)
+   * Get a single field from the given tableId and key tuple (loading schema from storage)
    */
-  function getField(uint256 table, bytes32[] memory key, uint8 schemaIndex) internal view returns (bytes memory) {
-    Schema schema = getSchema(table);
-    return getField(table, key, schemaIndex, schema);
+  function getField(uint256 tableId, bytes32[] memory key, uint8 schemaIndex) internal view returns (bytes memory) {
+    Schema schema = getSchema(tableId);
+    return getField(tableId, key, schemaIndex, schema);
   }
 
   /**
-   * Get a single field from the given table and key tuple, with the given schema
+   * Get a single field from the given tableId and key tuple, with the given schema
    */
   function getField(
-    uint256 table,
+    uint256 tableId,
     bytes32[] memory key,
     uint8 schemaIndex,
     Schema schema
   ) internal view returns (bytes memory) {
     if (schemaIndex < schema.numStaticFields()) {
-      return StoreCoreInternal._getStaticField(table, key, schemaIndex, schema);
+      return StoreCoreInternal._getStaticField(tableId, key, schemaIndex, schema);
     } else {
-      return StoreCoreInternal._getDynamicField(table, key, schemaIndex, schema);
+      return StoreCoreInternal._getDynamicField(tableId, key, schemaIndex, schema);
     }
   }
 }
 
 library StoreCoreInternal {
   bytes32 internal constant SLOT = keccak256("mud.store");
-  uint256 internal constant SCHEMA_TABLE = uint256(keccak256("mud.store.table.schema"));
+  uint256 internal constant SCHEMA_TABLE = uint256(bytes32(abi.encodePacked(bytes16(""), bytes16("schema"))));
```

Comment:
Important node @holic @authcall: this PR changes the ID of the schema table, which is relevant for the network package and MODE

Reply:
Does the empty namespace mean anything special? If not, do we wanna use a “mud” or “mud.internals” namespace for ourselves? Leaves the empty namespace for userland.

Reply:
The schema table is gonna be registered by the deployer of the world, so they'd own the `mud` namespace. Doesn't really matter though, happy to go with a mud specific namespace here

Reply:
updated namespace for all store tables to `mudstore`

Reply:
I like `mudstore` or similar just because errors/reverts will make a little more sense!